src/components/DashboardPage/DashboardPage.tsx

```diff
@@ -101,28 +110,38 @@ export const DashboardPage = ({ user, ssrTime, defaultPresetFallback }: External
                     counter={goalsCount}
                     filterPreset={preset}
                     loading={isLoading}
+                    enableLayoutToggle
                 />
             }
         >
             <ListView onKeyboardClick={handleItemEnter}>
-                {groupsOnScreen?.map(({ goals, ...project }) => (
-                    <ProjectListItemCollapsable
-                        key={project.id}
-                        interactive={false}
-                        visible
-                        project={project}
-                        href={routes.project(project.id)}
-                        goals={nullable(goals, (g) => (
-                            <TreeViewElement>
-                                <GoalTableList goals={g} />
-                            </TreeViewElement>
-                        ))}
-                    >
-                        {nullable(!goals?.length, () => (
-                            <InlineCreateGoalControl project={project} />
-                        ))}
-                    </ProjectListItemCollapsable>
-                ))}
+                {groupsOnScreen?.map(({ goals, ...project }) => {
+                    const canban = canbansByProject[project.id];
+
+                    const children =
+                        view === 'canban'
+                            ? nullable(canban, (value) => <Canban value={value} filterPreset={preset} />)
```

Comment:
useless `nullable` function's call 'cause `canban` is not array and cannot equals to null